src/openforms/forms/tests/test_api_formdefinition.py

```diff
@@ -156,6 +156,35 @@ def test_update(self):
         self.assertEqual(True, definition.login_required)
         self.assertIn({"label": "New field"}, definition.configuration["components"])
 
+    def test_update_is_reusable_unsuccessful_with_multiple_forms(self):
+        user = StaffUserFactory.create(user_permissions=["change_form"])
```

Comment:
permissions are not relevant for this test case, so you can use `SuperUserFactory` instead.